services.yaml

```diff
@@ -1464,6 +1630,28 @@ paths:
           $ref: "#/definitions/UnauthorizedError"
         404:
           description: NOT FOUND
+  /services/{service_id}/events/{event_id}/restart-snapshots:
```

Comment:
i think reset is better than restart and why not simply /reset?

Reply:
let's discuss this one

Reply:
updated to be force-snapshot